internal/data/metric.go

```diff
@@ -184,12 +179,17 @@ func (rm ResourceMetrics) InstrumentationLibraryMetrics() []InstrumentationLibra
 func (rm ResourceMetrics) SetInstrumentationLibraryMetrics(s []InstrumentationLibraryMetrics) {
 	rm.initInternallIfNeeded()
 	rm.pimpl.instrumentationLibraryMetrics = s
-	// We don't update the orig slice because this may be called multiple times.
-	rm.pimpl.sliceChanged = true
-}
 
-func (rm ResourceMetrics) getOrig() *otlpmetrics.ResourceMetrics {
```

Comment:
I cannot find getOrig() functions in the new code anymore? Are they deleted?

Reply:
Yes, accessing the orig directly it is easier :)